maximeish/MyBrand-Maxime
# landing page for brand website

#### What does this PR do?
Seeks approval for adding landing page


#### Description of Task to be completed?
Add a landing page to the brand website

#### How should this be manually tested?
Check if all the necessary HTML and CSS are present and the page should only fill the whole width and height of the viewport
i.e. no scrollbar should be present

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
N/A

#### Screenshots (if appropriate)
![landing page](https://user-images.githubusercontent.com/54126307/87796559-cf0b1800-c849-11ea-89b5-15cd96cbee20.png)


#### Questions:
Why am i seeing a merge error that says, "Can't automatically merge" ?
